token/core/cmd/pp/gen.go

```diff
@@ -10,82 +10,86 @@ import (
 	"bytes"
 	"encoding/base64"
 	"fmt"
-	"github.com/hyperledger-labs/fabric-token-sdk/token/core/fabtoken"
 	"io"
 	"io/ioutil"
 	"path/filepath"
 	"strings"
 	"text/template"
 
 	math3 "github.com/IBM/mathlib"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-
 	"github.com/hyperledger-labs/fabric-smart-client/integration/nwo/fabric/packager"
-
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric/core/generic/msp/x509"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/fabtoken"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/core/zkatdlog/crypto"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
 )
 
 const (
 	IdemixConfigDirMsp              = "msp"
 	IdemixConfigFileIssuerPublicKey = "IssuerPublicKey"
 )
 
-var driver string
-var idemixMSPDir string
-var output string
-var base int64
-var exponent int
-var cc bool
+var Driver string
+var IdemixMSPDir string
+var Output string
+var Base int64
+var Exponent int
+var CC bool
+var Issuers []string
+var Auditors []string
 
 // Cmd returns the Cobra Command for Version
 func Cmd() *cobra.Command {
 	// Set the flags on the node start command.
 	flags := cobraCommand.Flags()
-	flags.StringVarP(&driver, "driver", "d", "dlog", "driver (dlog, zkatdlog or fabtoken)")
-	flags.StringVarP(&idemixMSPDir, "idemix", "i", "", "idemix msp dir")
-	flags.StringVarP(&output, "output", "o", ".", "output folder")
-	flags.Int64VarP(&base, "base", "b", 100, "max token quantity")
-	flags.IntVarP(&exponent, "exponent", "e", 2, "max token quantity")
-	flags.BoolVarP(&cc, "cc", "", false, "generate chaincode package")
+	flags.StringVarP(&Driver, "driver", "d", "dlog", "driver (dlog, zkatdlog or fabtoken)")
+	flags.StringVarP(&IdemixMSPDir, "idemix", "i", "", "idemix msp dir")
+	flags.StringVarP(&Output, "output", "o", ".", "output folder")
+	flags.Int64VarP(&Base, "base", "b", 100, "max token quantity")
+	flags.IntVarP(&Exponent, "exponent", "e", 2, "max token quantity")
+	flags.BoolVarP(&CC, "cc", "", false, "generate chaincode package")
+	flags.StringSliceVarP(&Auditors, "auditors", "a", nil, "list of auditor keys")
```

Comment:
`flags.StringSliceVarP(&Auditors, "auditors", "a", nil, "list of auditor keys")`
`flags.StringSliceVarP(&Issuers, "issuers", "s", nil, "list of issuer keys")`

What is the expected format for this list?
Somewhere here we use `:` to split an auditor.